src/components/main_page/GongZhongHao.js

```diff
@@ -1,8 +1,46 @@
 import React from "react";
+//import "../../styles/main_page/_GongZhongHao.scss";
+//import "../../styles/styles/style.css.map";
 
 export default function GongZhongHao(props) {
     return (
-        <div className="gzh-container">
+        // <div className="gzh-container">
+        // </div>
+        <div className="MainGrid">
+            <article>
+                <a href="https://mp.weixin.qq.com/s?src=11&timestamp=1607286461&ver=2750&signature=jI8d4NQ08ctBdDO7eVHA0QKKAl6dhtO8Ip3QSr0PqnjQeG-YkH0PhDAx6sK00v5eAXBEv7PC8jdH0jZ0TW8g4wTCigfITtG0jxQuUfMynI*z2L47C1lXF8GcTGrsp9SR&new=1" class="button">
+                    <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/9/94/East_West_sushi_01.jpg/520px-East_West_sushi_01.jpg" alt="Sample photo"></img>
+                </a>
+                <div className="MainText">
+                    <br></br>
+                    <h5>多伦多日料外卖大赏</h5>
+                    <p>什么？在家也能吃到新鲜的日料？</p>
+                    
+                </div>
+            </article>
+            <article>
+            <a href="https://mp.weixin.qq.com/s?src=11&timestamp=1607298911&ver=2751&signature=ObJhORkkKeqABnyU7yhfMITKKqLSnh*ETWTtNHxOftJzjraTPRhmcRPBDbjUZoBd*swG4V4pGlW0bLEaq8cE*i19ntou1JotXwAJMYV6HqGLBSwDUGP6kPxn-lIwXSAK&new=1">
+                <img src="https://www.pearsoneducation.ca/wp-content/uploads/2019/09/shutterstock_116618521.jpg" alt="Sample photo"></img>
+            </a>
+                <div className="MainText">
```

Comment:
The naming convention for a class is `xxx-xxx-xxx`.

Reply:
https://github.com/UT-Applicataion-Development-Association/ADA-Website/pull/28/commits/bbcdde90e86d68ab6f2c7fffa7a41cbf79e487d3